scripts/docker/debian/Dockerfile

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+# Dockerfile that creates a container suitable to build dotnet-cli
+FROM debian:jessie
+
+# Misc Dependencies for build
+RUN apt-get update && apt-get -qqy install curl unzip gettext sudo
+
+# This could become a "microsoft/coreclr" image, since it just installs the dependencies for CoreCLR (and stdlib)
+RUN echo "deb http://llvm.org/apt/jessie/ llvm-toolchain-jessie-3.6 main"  | tee /etc/apt/sources.list.d/llvm.list && \
+    curl http://llvm.org/apt/llvm-snapshot.gpg.key | apt-key add - && \
+    apt-get update && apt-get -qqy install\
+	libc6 \
+	libedit2 \
+	libffi6 \
+	libgcc1 \
+	libicu52 \
+	liblldb-3.6 \
```

Comment:
@MichaelSimons 
I would like to remove it in the long run, because it requires some additional steps to install on debian Jessie. We'd need to look at how that impacts the debugging experience though.

I don't think it's going to be an issue until we make a debian package for the debian distro, so let's leave it for now.